README.md

```diff
@@ -28,18 +28,12 @@ Right now you can:
 
 ## Install
 
-Arch Linux users can just install the [lightdm-mini-greeter
-package][aur-package] from the Arch User Repository:
+Grab the source, build the greeter and install it manually:
```

Comment:
Leave the arch linux instructions here please, I'll add instructions for other distros if anyone ever makes packages for them.

Reply:
I removed it because I thought it was the git version and it needed to be updated to work again. My bad. 😓

Reply:
I can add Gentoo and Exherbo packages 👍

Reply:
That'd be awesome, I keep the arch linux package on the latest tag, but I could move new development to a `develop` branch & have `master` be the latest version if that helps you.

Reply:
They both use release tags, so there is no need for the `develop` branch. Thank you anyway :smile: